tests/unittests/OperatorTest.cpp

```diff
@@ -4925,6 +4925,52 @@ TEST_P(OperatorTest, FP16Max) {
   }
 }
 
+/// Helper to test Broadcast Max/Min using \p DTy.
+template <typename DataType>
+static void testBroadcastMaxMin(glow::PlaceholderBindings &bindings,
+                                glow::Module &mod, glow::Function *F,
+                                glow::ExecutionEngine &EE, ElemKind DTy,
+                                bool MaxOrMin) {
+
+  PseudoRNG PRNG;
+
+  auto *inputA = mod.createPlaceholder(DTy, {1, 3, 3, 1}, "A", false);
+  bindings.allocate(inputA)->getHandle<DataType>().randomize(-3.0, 3.0, PRNG);
+  auto *inputB = mod.createPlaceholder(DTy, {1, 3, 3, 1}, "B", false);
+  bindings.allocate(inputB)->getHandle<DataType>().randomize(-3.0, 3.0, PRNG);
+
+  Node *MaxorMinOp = nullptr;
+
+  if (MaxOrMin) {
```

Comment:
Alternative is just another template type for Node and just pass it in directlly.

Reply:
Done.